packages/contracts-bedrock/scripts/deploy/DeployConfig.s.sol

```diff
@@ -222,11 +222,6 @@ contract DeployConfig is Script {
         useAltDA = _useAltDA;
     }
 
-    /// @notice Allow the `useFaultProofs` config to be overridden in testing environments
-    function setUseFaultProofs(bool _useFaultProofs) public {
-        useFaultProofs = _useFaultProofs;
```

Comment:
we should be able to delete the `useFaultProofs` storage as well. Unless that breaks some other script that expects it.

Reply:
good call, done.